src/IngredientEntity.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace GRUB; 
+
+/* Class for IngredientEntities */
```

Comment:
don't think you need this, pretty self explanatory

Reply:
Ashley said yes we do

Reply:
shut my mouth then